prereise/gather/griddata/hifld/data_process/generators.py

```diff
@@ -132,5 +188,11 @@ def build_plant(bus, substations):
         ["Summer Capacity (MW)", "Winter Capacity (MW)"]
     ].max(axis=1)
     generators.rename({"Minimum Load (MW)": "Pmin"}, inplace=True, axis=1)
+    print("Fitting heat rate curves to EPA data... (this may take several minutes)")
+    heat_rate_curve_estimates = generators.apply(
+        lambda x: estimate_heat_rate_curve(x, epa_ampd_groupby, crosswalk_translation),
+        axis=1,
+    )
```

Comment:
Should we have a way to use the `min_x_points` optional parameter through `build_plant` as we do with the optional `rounding` parameter in `build_transmission`

Reply:
Done.